announcements/2020-Week-49/announcement.md

```diff
@@ -0,0 +1,11 @@
+# :zap: This Week in Provider-Independent Announcements (December 04, 2020):zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed Apps
+
+1. [Kong v1.1.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#110---2020-12-02) provides kong version 2.2.
+1. [cert-exporter v1.4.0](https://github.com/giantswarm/cert-exporter/blob/master/CHANGELOG.md#140---2020-12-02) adds a new metric which tracks expiry of certificates in Kubernetes secrets.
```

Comment:
@ubergesundheit @glitchcrab cert-exporter is not an optional app and would therefore be included in a tenant cluster release, right? If so, we should not include here 😀